code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -74,6 +74,51 @@
 /obj/item/weapon/cautery/is_hot()
 	return 1
 
+/obj/item/weapon/cautery/laser
+	sharpness = 1.5
+	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
+	damtype = "fire"
+	force = 10.0
+	throwforce = 5.0
+	throw_speed = 3
+	throw_range = 5
+	attack_verb = list("attacks", "slashes", "stabs", "slices", "tears", "rips", "dices", "cuts")
+	hitsound = "sound/weapons/bladeslice.ogg"
+	var/flip = /obj/item/weapon/scalpel/laser
+
+/obj/item/weapon/cautery/laser/attack_self(mob/user)
+	var/old_loc = loc
+	if(loc == user)
+		user.drop_from_inventory(src)
+	else if(ismob(loc))
+		var/mob/holder = loc
+		holder.drop_from_inventory(src)
+	if(flip)
+		if(ispath(flip, /obj/item))
+			var/obj/item/FlipItem = new flip(old_loc)
+			if(ismob(old_loc))
+				var/mob/M = old_loc
+				M.put_in_hands(FlipItem)
+	to_chat(user, "You return the scalpel to cutting mode.")
+	qdel(src)
```

Comment:
Swapping items back and forth by repeatedly deleting one and spawning the other is bad practice.